content/docs/command-reference/remote/add.md

```diff
@@ -104,7 +104,7 @@ By default, DVC authenticates using your AWS CLI
 (if set). This uses the default AWS credentials file. To use a custom
 authentication method, use the parameters described in `dvc remote modify`.
 
-S3 remotes can also be configured entirely via environment variables, e.g.:
+S3 remotes can also be configured via environment variables, for example:
 
 ```dvc
 $ export AWS_ACCESS_KEY_ID='mykey'
```

Comment:
> Moving from https://github.com/iterative/dvc.org/pull/2415#discussion_r626199728, from @shcheklein:

A think a lot of remotes can be configured with env vars.

All this additional config should be part of modify to my mind. Things like

```
export ..
export
dvc remote add
```

do not make much sense overall. 

```
export ...
export ...
```

are important when you run `dvc push`/`dvc pull`. It's a way to provide values to auth flow used in DVC when it tries to access. Other ways - default config files (`.aws/credentials`) or DVC config (the one you modify with `dvc remote modify`)

Reply:
⌛

Reply:
> a lot of remotes can be configured with env vars

Good Q. Opened https://github.com/iterative/dvc.org/issues/2445

> export & dvc remote add ... do not make much sense

Wait yeah... `remote add` doesn't need any creds. OK, removed all env var talk from that ref.

Reply:
> a lot of remotes can be configured with env vars

Good Q @shcheklein . Opened https://github.com/iterative/dvc.org/issues/2445

> export & dvc remote add ... do not make much sense

Wait yeah... `remote add` doesn't need any creds. OK, removed all env var talk from that ref.

> are important when you run dvc push/dvc pull

True. Updated `remote modify`. PTAL